src/App.tsx

```diff
@@ -1,24 +1,158 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import {
+  USER_ID,
+  createTodo,
+  destroyTodo,
+  getTodos,
+} from './api/todos';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoInfo } from './components/TodoInfo/TodoInfo';
 
-const USER_ID = 0;
+type IsCompleted = 'all' | 'active' | 'completed';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [completedFilter, setCompletedFilter] = useState<IsCompleted>('all');
+  const [isEmptyTitleError, setIsEmptyTitleError] = useState(false);
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [isCreatingError, setIsCreatingError] = useState(false);
+  const [isDeletedError, setIsDeletedError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isUploadingTitle, setIsUploadingTitle] = useState(false);
+  const [isLoadingTodoId, setIsLoadingTodoId] = useState<number | null>(null);
```

Comment:
it's not a `boolean` type
```suggestion
  const [loadingTodoId, setLoadingTodoId] = useState<number | null>(null);
```